conf/server.properties

```diff
@@ -28,8 +28,8 @@ org.dbpedia.spotlight.language = English
 org.dbpedia.spotlight.language_i18n_code = en
 # Stop word list
 # An example can be downloaded from: http://spotlight.dbpedia.org/download/release-0.4/stopwords.en.list
-org.dbpedia.spotlight.data.stopWords.english = stopwords.en.list
-org.dbpedia.spotlight.data.stopWords.portuguese = /usr/local/spotlight/dbpedia_data/data/stopwords.pt.list
+org.dbpedia.spotlight.data.stopWords.english = data_quickstart/stopwords.en.list
```

Comment:
Sure, but afaik the whole server.properties file is only really applicable when the lucene backend is used, right?